SEED-platform/seed
# Add Building Sync audit template id and pm_property_id to column mappings

#### What's this PR do?
* Add new BSync fields to the default BSync column mapping profile.

#### How should this be manually tested?
* run migration
* load a BuildingSync file
* verify that the pm property id and audit template id mappings have defaulted to appropriate fields

#### What are the relevant tickets?
n/a

#### Screenshots (if appropriate)
<img width="1190" alt="image" src="https://user-images.githubusercontent.com/1907354/174139365-a3231f30-2414-48c9-a3d6-9040c2ffcedb.png">
